src/utils/keyRotation.ts

```diff
@@ -0,0 +1,13 @@
+import { Bindings } from '../bindings';
+
+export function shouldRotateKey(date: Date, env: Bindings): boolean {
+	const dayOfMonth = parseInt(env.ROTATION_DAY_OF_MONTH, 10);
+	const dayOfWeek = parseInt(env.ROTATION_DAY_OF_WEEK, 10);
+	const hourOfDay = parseInt(env.ROTATION_HOUR_OF_DAY, 10);
```

Comment:
rotation might happen multiple times in a week. It would be interesting to add a wildcard